updates/seed_ir_states.php

```diff
@@ -0,0 +1,46 @@
+<?php namespace RainLab\Location\Updates;
+use October\Rain\Database\Updates\Seeder;
+use RainLab\Location\Models\Country;
+class SeedIrStates extends Seeder
```

Comment:
Add empty lines after the first `namespace` line and before the `class` definition line